bellydynamic-metrics/Centrality.py

```diff
@@ -81,31 +81,31 @@ def getOutputFileName(self, suffix):
         return self.graphName + suffix + ".txt"
 
     def genGraphInfoBetweeness(self, centrality):
-        print "\n+++++++++++ %s ++++++++++++" % (centrality)
-        print "Calculating centrality values..."
+        print ("\n+++++++++++ %s ++++++++++++" % (centrality))
```

Comment:
Seems you're converting the code to Python 3, better to use the Python3 recommended formatting style with `{}`.

Reply:
Sorry for the delay, but if I may ask, can you clarify what you mean by using a formatting style with   { }? Do you mean for me to further format the print statements using { }? Or the code in general? Also, if you are looking to transform all the code in the application into Python 3, may I suggest using "2to3", a Python tool for converting such code?